app/notifications/process_letter_notifications.py

```diff
@@ -0,0 +1,44 @@
+from app import create_random_identifier
+from app.models import LETTER_TYPE, JOB_STATUS_READY_TO_SEND, Job
+from app.dao.jobs_dao import dao_create_job
+from app.notifications.process_notifications import persist_notification
+from app.v2.errors import InvalidRequest
+
+
+def create_letter_api_job(template):
+    service = template.service
+    if not service.active:
+        raise InvalidRequest('Service {} is inactive'.format(service.id), 403)
+    if template.archived:
+        raise InvalidRequest('Template {} is deleted'.format(template.id), 400)
+
+    job = Job(
+        original_file_name='letter submitted via api',
+        service=service,
```

Comment:
we need to give it a name we can easily filter out so that it doesn't show up on the admin 😩